remote-trait-object/src/transport.rs

```diff
@@ -30,6 +30,7 @@ pub trait TransportSend: Sync + Send + std::fmt::Debug {
 pub enum RecvError {
     TimeOut,
     Termination,
+    Closed,
```

Comment:
How are `Closed` and `Termination` different?

Reply:
`Termination` -> triggered by `Terminate`
`Closed` -> The other end has closed the connection

Reply:
I'll make it as `Custom` for now